lib/karafka/errors.rb

```diff
@@ -52,5 +52,8 @@ class TopicMultipleUsage < BaseError; end
 
     # Raised when we didn't use a topic that was defined as non-optional (required)
     class UnusedResponderRequiredTopic < BaseError; end
+
+    # Raised when configuration doesn't match with validation schema
+    class WrongConfiguration < BaseError; end
```

Comment:
Please change to Karafka::Errors::InvalidConfiguration - we use for other errors the "Invalid" prefix so it would be good for consistency to keep it that way.